### PR TITLE
V4 to V5 AEP Analytics migration

### PR DESCRIPTION
**Pull request by rymorale**

### CODE REVIEW

File: resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
+# Migrating from V4 Analytics to the AEP Analytics Extension
+
+## Configuration
+
+The AEP Analytics extension uses [Launch](https://launch.adobe.com/) to configure the AEP SDK's. This replaces the ADBMobileConfig.json which the V4 SDK used for configuration. To get started with the AEP SDK's:
+
+1. Create a mobile property on Launch. See [Set up a mobile property](../../getting-started/create-a-mobile-property) for more information.
+2. Configure your mobile app with the create mobile property. The AEP Mobile Core extension provides general functionality required by all the Adobe AEP extensions. The Configuration extension is buult into the Mobile Core and contains the `configureWithAppId` API. This API is used to link the Launch mobile property with your mobile app. The documentation for this API can be seen at the [Configuration API Reference](../../using-mobile-extensions/mobile-core/configuration/configuration-api-reference#configurewithappid) page. A code sample showing the usage of this API is provided below.
```
Comment (cacheung) [inline resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md:8]: Typo "buult"

File: resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+| trackTimedActionUpdate: ([iOS](https://marketing.adobe.com/resources/help/en_US/mobile/ios/timed_actions.html), [Android](https://marketing.adobe.com/resources/help/en_US/mobile/android/timed_actions.html)) | This functionality can be recreated using the [Analytics](../../using-mobile-extensions/adobe-analytics) and [User Profile](../../using-mobile-extensions/profile) extensions. |
+| trackTimedActionEnd: ([iOS](https://marketing.adobe.com/resources/help/en_US/mobile/ios/timed_actions.html), [Android](https://marketing.adobe.com/resources/help/en_US/mobile/android/timed_actions.html)) | This functionality can be recreated using the [Analytics](../../using-mobile-extensions/adobe-analytics) and [User Profile](../../using-mobile-extensions/profile) extensions. |
+| trackTimedActionExists: ([iOS](https://marketing.adobe.com/resources/help/en_US/mobile/ios/timed_actions.html), [Android](https://marketing.adobe.com/resources/help/en_US/mobile/android/timed_actions.html)) | This functionality can be recreated using the [Analytics](../../using-mobile-extensions/adobe-analytics) and [User Profile](../../using-mobile-extensions/profile) extensions. |
+
+#### API Changes
```
Comment (cacheung) [inline resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md:28]: Below is more like initializing or setup for the extension.  Not really API change

File: resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+  }
+  return true
+}
+```
+
+{% endtab %}
+{% endtabs %}
+
+In-depth instructions can be seen at the [Analytics Readme](../../using-mobile-extensions/adobe-analytics/readme.md#add-analytics-to-your-app).
+
```
Comment (cacheung) [inline resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md:110]: Below can be API changes

File: resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+// usage
+MobileCore.trackAction("linkClicked", new HashMap<String, String>() {{
+  put("url", "https://www.adobe.com");
+}});
+```
+
+{% endtab %}
+
+{% tab title="iOS" %}
+
```
Comment (cacheung) [inline resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md:163]: Is this For V4? Not sure how tabs will look like, will the users know this part is for V4?

File: resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md [hunk 5]
```diff
@@ -0,0 +1,315 @@
...
+// syntax
+public static void trackAction(final String action, final Map<String, Object> contextData)
+  
+// usage
+Analytics.trackAction("linkClicked", new HashMap<String, Object>() {{
+  put("url", "https://www.adobe.com");
+}});
+```
+
+The AEP SDK's have moved the `trackAction` and `trackState` APIs to the MobileCore extension. In addition, the context data Map has been changed from `<String, Object>` to `<String, String>`:
```
Comment (cacheung) [inline resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md:138]: Feel like need to emphasis this section for AEP SDK implementation so user can look at this directly.  May be bold text or highlight?

File: resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md [hunk 6]
```diff
@@ -0,0 +1,315 @@
...
+
+```objective-c
+// syntax
++ (void) trackAction:(NSString *)action data:(NSDictionary *)data;
+
+// usage
+[ADBMobile trackAction:@"linkClicked" data:@{@"url":@"https://www.adobe.com"}];
+```
+
+The AEP SDK's have moved the `trackAction` and `trackState` API's to the MobileCore extension. In addition, the NSDictionary has been changed from `<NSString, NSObject>` to `<NSString, NSString>`:
```
Comment (cacheung) [inline resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md:180]: Feel like need to emphasis this section for AEP SDK implementation so user can look at this directly.  May be bold text or highlight?

**Comment by rymorale** [comment]
Hi @imironchuk, would you be able to merge this doc update in?

**Pull request merged**